ga4gh/client.py

```diff
@@ -29,7 +54,89 @@ def __init__(self, urlPrefix, debugLevel, workarounds, key):
         self._workarounds = workarounds
         self._key = key
 
-    def runRequest(self, request, url, protocolClass, listAttr):
+    # TODO temporary auth solution
+    def _getAuth(self):
+        if self._usingWorkaroundsFor(self.workaroundGoogle):
+            return {'key': self._key}
+        else:
+            return {}
+
+    def _usingWorkaroundsFor(self, workaround):
+        return workaround in self._workarounds
+
+    def _debugResponse(self, jsonString):
+        if self._debugLevel > 1:
+            # TODO use a logging output and integrate with HTTP client more
+            # nicely... also, logging / error handling in general
+            print("json response:")
+            pp = json.dumps(
+                json.loads(jsonString), sort_keys=True, indent=4)
+            print(pp)
+
+    def _checkStatus(self, response):
+        if response.status_code != 200:
```

Comment:
It would be nice to use `requests.codes.ok` here instead of 200. See http://docs.python-requests.org/en/latest/user/quickstart/#response-status-codes